knowledge_repo/converters/rmd.py

```diff
@@ -21,6 +21,9 @@ def from_file(self, filename, rebuild=True):
                         x = knit('{1}', '{2}', quiet=T)" """.format(os.path.abspath(os.path.dirname(filename)),
                                                                     os.path.abspath(filename),
                                                                     tmp_path)
+            if os.name == 'nt':
+                runcmd = runcmd.replace("\\", "\\\\")                                                        
```

Comment:
You have excess whitespace to the left of `runcmd.replace("\\", "\\\\")`. Once you remove it, the build should pass and we can merge this change in.